python/constraints.txt

```diff
@@ -165,14 +159,12 @@ et-xmlfile==1.1.0
     # via openpyxl
 executing==2.0.1
     # via stack-data
-faker==25.9.1
+faker==25.9.2
     # via -r python/requirements.in
 filelock==3.15.4
     # via
     #   gdown
     #   virtualenv
-fiona==1.9.6
```

Comment:
Intriguing

Reply:
this is probably due `geopandas==1.0.0`? and good news? lol

Reply:
my guess is geopandas changing their default engine for geo files from `fiona` to `pyogrio`...

Reply:
yep,[ it changed](https://geopandas.org/en/latest/docs/user_guide/fiona_to_pyogrio.html#) and it's supposed to be faster.

<img width="767" alt="image" src="https://github.com/NYCPlanning/data-engineering/assets/144725249/951f7ff8-50b0-43ab-8cd5-c5a5f76409a1">